metabase/metabase
# Hide model cache monitoring page when caching is turned off

#21551 added a simple page to monitor models cache refreshes to Admin > Tools (making the page partially available in OSS). It doesn't make a lot of sense to show the page when people do not use model persistence. This PR hides the page if model persistence is not enabled on an instance.

### To Verify

1. Sign in as an admin
2. Run Metabase OSS
3. Go to `/admin/settings/caching` and turn off model persistence ("Models" toggle)
4. Ensure you can't see the "Tools" navigation item in the admin navigation bar
5. Go back to `/admin/settings/caching`, enable model persistence
6. Ensure you can see and access the "Tools" page and you can only see a table with model cache refreshes statuses
7. Run Metabase EE
8. Repeat the same steps, but make sure "Tools" are always visible and you just can see "Model Caching Logs" tab appear/disappear

### Demo

<details>
<summary>OSS</summary>

https://user-images.githubusercontent.com/17258145/163860895-56b44db8-cf68-4035-8ab8-1fae4003441e.mp4

</details>

<details>
<summary>EE</summary>

https://user-images.githubusercontent.com/17258145/163860906-7c682945-a9b1-439a-9660-f36450878f28.mp4

</details>
